setup.py

```diff
@@ -76,7 +76,7 @@ def find_version(*file_paths):
     install_requires=requirements,
     extras_require={
         'extras': [
-            'boto3',
+            'boto3==1.14.32',
```

Comment:
How about specifying an acceptable arrange, as this affects all users installing with `[extra]`

Reply:
`awscli` and `boto3` all require `botocore` this dependency. `botocore 1.18.0` has removed doc files and migrated to `awscli >= 1.18.140`. Since we installed [`awscli`](https://github.com/dmlc/gluon-nlp/blob/56de303f2283937cca5c0e67703713d624b6fd05/tools/batch/docker/Dockerfile.gpu#L29)during building the docker file and installed [`boto3`](https://github.com/dmlc/gluon-nlp/blob/56de303f2283937cca5c0e67703713d624b6fd05/tools/batch/docker/gluon_nlp_job.sh#L23) when boot up an AWS batch job. So, to solve docevents missing issue, we can either keep `boto3` and `awscli` all in old version or update `awscli` by updating the docker and take `boto3` in newest version.